src/main/scala/ch/epfl/bluebrain/nexus/admin/organizations/OrganizationRejection.scala

```diff
@@ -1,6 +1,22 @@
 package ch.epfl.bluebrain.nexus.admin.organizations
+
 import ch.epfl.bluebrain.nexus.commons.types.Rejection
 
 sealed trait OrganizationRejection extends Rejection
```

Comment:
please drop the use of the stuff in common types, just use:
`sealed trait OrganizationRejection extends Product with Serializable`